services/motion/builtin/validation_test.go

```diff
@@ -315,12 +315,13 @@ func TestMoveCallInputs(t *testing.T) {
 					Destination:   goalPose,
 					SlamName:      slam.Named("test_slam"),
 					MotionCfg:     &motion.MotionConfiguration{PlanDeviationMM: 10},
-					Extra:         map[string]interface{}{"collision_buffer_mm": 200.},
+					Extra:         map[string]interface{}{"collision_buffer_mm": 2000.},
```

Comment:
this is a humongous jump. From 20cm to 2 meters... This worries me.

Reply:
The old test was never working

Reply:
If I go on main and run this test with
```
go test -run TestMoveCallInputs/MoveOnMap/collision_buffer_mm_validtations/fail_when_collisions_are_predicted_within_the_collision_buffer -v
```
it passes. So, "The old test was never working" is not true.

Reply:
"Passing" does not mean it was doing the correct thing.  Basically any collision buffer >1 would have "passed" this test as its currently written.  When in reality there should have been no collision.  When I fixed the bugs this PR is addressing this test started failing becuase there was in fact no collision.  In order for there to be a collision I needed to make the buffer larger

Reply:
amazing, thank you for clarifying.

Reply:
I also don't understand how choosing an arbitrarily large number would even be bad here.  All we want to see is that we correctly error on collision.  So choosing MaxInt would also be an acceptable number here

Reply:
> All we want to see is that we correctly error on collision. So choosing MaxInt would also be an acceptable number here

That's true.
I was operating under the assumption that the smallest satisfying number would be appropriate for a test like this since we want to show that it doesn't only work in the trivial case where we give it a 2m collision buffer.